docker/docker-build-ee.bat

```diff
@@ -0,0 +1,19 @@
+@echo off
+echo.
+:: Adapted from Geoff Boeing utility script for building a single platform Docker image
+::https://github.com/gboeing/osmnx/blob/2631b95a28300ccffce0c7c6838d269ce708ec1a/environments/docker/docker-build-single_platform.sh
+set "DOCKERUSER=ryntrnr"
```

Comment:
should be our organisation name as in docker-build. bat, not your name for this pull request purpose